templates/portal/apps.html

```diff
@@ -7,87 +7,119 @@
 
     <div class="row">
         <div class="col">
-            <h3 class="mb-3" style="margin-right: 5px;">
+            <h3 class="h3 mb-3 me-5">
                 Public apps
                 {% if published_apps|length > 0 %}
                 <span class="ghost-number">{{ published_apps|length }}</span>
                 {% endif %}
 
             </h3>
 
-            <div>
+            <div class="tag-list">
                 {% for tf in request.session.app_tag_filters %}
-                <div class="tag tag-list"><a class="tag-list-ico fas fa-times" style="color: #fffafa;"
-                        href="{% url 'portal:index' %}?tf_remove={{ tf | urlencode }}"></a><span style="font-weight: 500;">{{ tf
-                        }}</span></div>
+                <div class="tag d-inline">
+                    <span class="badge text-bg-primary">{{ tf }}
+                        <a class="tag-list-ico fas fa-times" style="color: inherit;"
+                        href="{% url 'portal:apps' %}?tf_remove={{ tf | urlencode }}">x</a>
+                    </span>
+                </div>
                 {% endfor %}
             </div>
         </div>
     </div>
-<div class="row pt-3 g-4">
-    {% for app in published_apps %}
-    <div class="col-12 col-md-6 col-lg-6 col-xl-4">
-        <div class="card h-100 border-0 shadow">
 
-            <div style="height:350px;" class="align-items-center d-flex justify-content-center p-2">
+    <div class="row pt-3 g-4">
+        <div class="card-group">
+            <div class="row row-cols-1 row-cols-md-3 g-4">
+                {% for app in published_apps %}
+                <div class="col-12 col-sm-12 col-md-6 col-lg-6 col-xl-4 col-xxl-4 mb-4 pb-4">
+                    <div class="card h-100 border-0 shadow" style="min-width: 300px;">
+                        <div class="card-body d-flex flex-column mx-4 my-1">
+                            <div class="col-12 d-flex flex-row pb-2 align-items-center">
+                                <div class="col-8 d-flex">
+                                    <h5 class="card-title bg-white">{{ app.name }}</h5>
+                                </div>
+                                <div class="col-4 d-flex justify-content-end">
+                                    {% static 'images/logos/apps/' as static_url %}
+                                    <img class="img-fluid float-end img-cover" style="height:50px;padding-right: 5%;"
+                                        src="{{static_url}}{{app.app.slug}}-logo.png"
```

Comment:
I think this is not the same as used in the helm charts for installing the apps. For instance, the `Standard App` uses `default-logo.png`. This needs to be solved somehow. Perhaps it is possible to fetch the "logo" information from the installed app in the database?